src/main/webapp/javascript/search.js

```diff
@@ -34,7 +34,15 @@ function redirectSearchPage() {
 /**
```

Comment:
Are this and MainPage.jsp from userQueryUpdate?

Reply:
I can step in here, but I'm not entirely sure I remember. UserQueyUpdate was more focused on the updates to the servlet and such, but there was a much smaller PR earlier today (Friday 31, July) that was search redirecting